### PR TITLE
V3.8

### PR DESCRIPTION
**Pull request by System-Glitch**

### CODE REVIEW

File: src/guide/advanced/authentication.md [hunk 1]
```diff
@@ -200,16 +208,186 @@ To request a protected route, you will need to add the following header:
...
 authenticator := auth.Middleware(&model.User{}, &auth.JWTAuthenticator{Optional: true})
 router.Middleware(authenticator)
 ```
 
+##### Custom ID claim name
+
+By default, `auth.JWTAuthenticator` looks for the claim named `userid` in the given token. You can customize the name of the token using the `ClaimName` field:
+
+``` go
+authenticator := auth.Middleware(&model.User{}, &auth.JWTAuthenticator{Optional: true})
```
Comment (ethicnology) [inline src/guide/advanced/authentication.md:225]: It seems this example is a copy/past from the previous category.
I think it missing the struct example which show how to use ClaimName, am i right ?

File: src/guide/advanced/authentication.md [hunk 2]
```diff
@@ -200,16 +208,186 @@ To request a protected route, you will need to add the following header:
...
+##### Optional authentication
+
 This provider supports the `Optional` flag, which defines if the authenticator allows requests that don't provide credentials. Handlers should therefore check if `request.User` is not `nil` before accessing it.
 
 ``` go
 authenticator := auth.Middleware(&model.User{}, &auth.JWTAuthenticator{Optional: true})
 router.Middleware(authenticator)
 ```
 
+##### Custom ID claim name
```
Comment (ethicnology) [inline src/guide/advanced/authentication.md:220]: These titles are very small in the theme in comparison of h4

**Review by ethicnology — COMMENTED** [review]
I think you should improve the example from this section  **Custom ID claim name**, which is the same example as **Optional authentication** section.

File: src/guide/advanced/authentication.md (hunk 1 above)
Comment (System-Glitch) [reply src/guide/advanced/authentication.md:225]: Good catch, thank you!

File: src/guide/advanced/authentication.md (hunk 2 above)
Comment (System-Glitch) [reply src/guide/advanced/authentication.md:220]: They have `1rem` size, have bold font and have some margin. I think they look large enough.

**Pull request merged**